.changeset/olive-apes-battle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fake-scope/fake-pkg": patch
+---
+
+playwright testing addded for features to make sure nothing breaks as the project grows
```

Comment:
This file shouldn't be placed here, it should go into `templates/default/.changeset`